lib/breakers/statsd_plugin.rb

```diff
@@ -5,22 +5,7 @@ class StatsdPlugin
     def get_tags(request)
       tags = []
       if request
-        if request.url&.path
-          # replace identifiers with 'xxx'
-          # this nasty-looking regex attempts to cover:
-          # * (possibly negative) digit identifiers
-          # * uuid's with or without dashes
-          # * institution id's of form 111A2222 or 11A22222
-          digit = /\-?\d+/
-          contact_id = /\d{10}V\d{6}(%5ENI%5E200M%5EUSVHA)*/
-          uuids = /[a-fA-F0-9]{8}(\-?[a-fA-F0-9]{4}){3}\-?[a-fA-F0-9]{12}/
-          institution_ids = /[\dA-Z]{8}/
-          provider_ids = /Providers\(\d{10}\)/
-          r = %r{(\/)(#{digit}|#{contact_id}|#{uuids}|#{institution_ids}|#{provider_ids})(\/|$)}
-          endpoint = request.url.path.gsub(r, '\1xxx\5')
-          tags.append("endpoint:#{endpoint}")
-        end
-
```

Comment:
The 2 new regex's i added broke the lint for maximum lines, so I moved this into a private method below.